src/game/scenes/Fight.js

```diff
@@ -47,7 +49,8 @@ export class Fight extends Phaser.Scene {
     preload() {
         this.load.audio(
             "runRiot",
-            "../../../public/assets/sound/run-riot-matt-stewart-evans-main-version-02-03-14904.mp3"
+            import.meta.env.BASE_URL +
+                "../../../public/assets/sound/run-riot-matt-stewart-evans-main-version-02-03-14904.mp3"
```

Comment:
would we change the relative import to a partial url (for lack of a better term)?

the following is our base url for production and I think a relative url would not work as it would be appended to what is below:
`export default defineConfig({
    base: "/alien-invasion-game/",`